### PR TITLE
fix: Log more details on http errors

### PR DESCRIPTION
**Pull request by mkmik**

### CODE REVIEW

File: src/influxdb_ioxd/http.rs [hunk 1]
```diff
@@ -373,19 +373,18 @@ where
 // The API-global error handler, handles ApplicationErrors originating from
 // individual routes and middlewares, along with errors from the router itself
 async fn error_handler(err: RouterError<ApplicationError>, req: RequestInfo) -> Response<Body> {
+    let method = req.method().clone();
+    let uri = req.uri().clone();
+    let span_id = req.headers().get("x-b3-spanid");
```
Comment (alamb) [inline src/influxdb_ioxd/http.rs:378]: I don't know what `x-b3-spanid` means but I assume you know what you are doing

Comment (alamb) [reply src/influxdb_ioxd/http.rs:378]: I wonder if logging client / referer (eventually) might gain some other insight

**Review by alamb — APPROVED** [review]

File: src/influxdb_ioxd/http.rs (hunk 1 above)
Comment (mkmik) [reply src/influxdb_ioxd/http.rs:381]: The tracing span ID is unique for a whole "conversation". It allows us to correlate this error with other logs captured in other places, such as envoy (which can log both the full incoming http request and the outgoing grpc requests)

Comment (alamb) [reply src/influxdb_ioxd/http.rs:381]: Good point.

**Pull request merged**